### PR TITLE
Rework the widgets that still use deprecated addAnswerView method

### PR DESCRIPTION
**Pull request by grzesiek2010**
#### Why is this the best possible solution? Were any other approaches considered?
A long time ago we started reworking the way we build layouts for our widgets. There were some widgets left so I've decided to finish this process and rework them too. Since this is the continuation of what we started long time ago as I said there was no other option to consider. As it was described in https://github.com/getodk/collect/pull/5886/files#diff-4ece0412c697506a06de7e67214b39a19fe5ec38fbe1db0c23cfc635aea3a964L310 `onCreateAnswerView` should be used instead of `addAnswerView` method. Additionally, I've also reworked some widgets building their layouts using XML instead of programmatically (image widgets is a good example) to make adding paddings/margins easier and consistent with other widgets.

#### How does this change affect users? Describe intentional changes to behavior and behavior that could have accidentally been affected by code changes. In other words, what are the regression risks?
The list of refactored widgets contains:
- the image widgets (image/draw/signature/annotate)
- the ranking widget
- the video widget
- the likert widget
- the ExPrinterWIdget
- the ExStringWidgets
- text widgets (text/number etc.)

Those widgets should be tested for regression. Nothing should change they are just reworked internally.

#### Do we need any specific form for testing your changes? If so, please attach one.
No.

#### Does this change require updates to documentation? If so, please file an issue [here]( https://github.com/getodk/docs/issues/new) and include the link below.
No.

#### Before submitting this PR, please make sure you have:
- [x] added or modified tests for any new or changed behavior
- [x] run `./gradlew connectedAndroidTest` (or `./gradlew testLab`) and confirmed all checks still pass
- [x] added a comment above any new strings describing it for translators
- [x] verified that any code or assets from external sources are properly credited in comments and/or in the [about file](https://github.com/getodk/collect/blob/master/collect_app/src/main/assets/open_source_licenses.html).
- [x] verified that any new UI elements use theme colors. [UI Components Style guidelines](https://github.com/getodk/collect/blob/master/docs/CODE-GUIDELINES.md#ui-components-style-guidelines)

### CODE REVIEW

File: collect_app/src/main/res/layout/annotate_widget.xml [hunk 1]
```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <org.odk.collect.androidshared.ui.multiclicksafe.MultiClickSafeButton
+        android:id="@+id/capture_button"
+        style="?materialButtonStyle"
```
Comment (seadowg) [inline collect_app/src/main/res/layout/annotate_widget.xml]: This (and the other buttons in this PR) should use `?widgetButtonQuestionWidgetStyle` I think?

**Review by seadowg — CHANGES_REQUESTED** [review]
Loving this! I've spotted that we only have one use left of `createSimpleButton` (in `ExStringWidget`). Could we get rid of that as well?

File: collect_app/src/main/res/layout/annotate_widget.xml (hunk 1 above)
Comment (grzesiek2010) [reply collect_app/src/main/res/layout/annotate_widget.xml]: Right, fixed.

**Comment by grzesiek2010** [comment]
> Loving this! I've spotted that we only have one use left of createSimpleButton (in ExStringWidget). Could we get rid of that as well?

Done.

**Review by seadowg — APPROVED** [review]

**Comment by dbemke** [comment]
**Tested with Success!**

Verified on device with Android 10

**Verified cases:**
- buttons in widgets
- regression checks in the image widgets,the ranking widget,the video widget, the likert widget,text widgets

**Comment by srujner** [comment]
**Tested with Success!**

Verified on device with Android 13

**Pull request merged**